pyproject.toml

```diff
@@ -89,6 +89,11 @@ myst-parser = "^0.17.0"
 sphinx-pyproject = "^0.1.0"
 nbsphinx = "^0.8.8"
 
+[tool.poetry.group.dev.dependencies]
+makim = "1.6.4"
```

Comment:
shouldn't these versions start with a "^"?

Reply:
we are using version 1 as a dev version, because semantic release doesn't accept version 0.
so for now it would be better to have it hard pinned until we achieve  version 2.

Reply:
Ok, so for me  its looking good overall.

Reply:
awesome! thanks for the review!